build.gradle

```diff
@@ -41,6 +45,9 @@ dependencies {
     // Dependency injection
     compile 'com.google.inject:guice:4.0'
 
+    // Plugin meta
+    compile 'org.spongepowered:plugin-meta:0.1-SNAPSHOT'
```

Comment:
It's used for writing the `mcmod.info` file (here: from the annotation processor), and also used in the Gradle plugin.